### DISCUSSION TITLE
certificate verify failed: unable to get local issuer certificate

### DISCUSSION THREAD

**Discussion by jvam90** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#292] [Q&A] [answered]
I've been trying to use the udemy enroller with my mac and when i start it, i get this "certificate verify failed: unable to get local issuer certificate" error from coursevania, discudemy and tutorialbar. Is there any way to fix this?

**Comment by fakeid30** [comment, marked as answer]
The error is unclear. Please give instruction to reproduce.
